test/e2e/libpod_suite_test.go

```diff
@@ -629,6 +629,18 @@ func IsCommandAvailable(command string) bool {
 	return true
 }
 
+// WriteJsonFile write json format data to a json file
+func WriteJsonFile(data []byte, filePath string) bool {
+	var jsonData map[string]interface{}
+	json.Unmarshal(data, &jsonData)
+	formatJson, _ := json.MarshalIndent(jsonData, "", "	")
+	err := ioutil.WriteFile(filePath, formatJson, 0644)
+	if err != nil {
+		return false
```

Comment:
It would probably be good to have at least an informational message here on error.

Reply:
Thanks for comments. Will return the err directly in this function. And let the test cases to decide to how to deal with it.